AmazonChimeSDK/AmazonChimeSDK/audiovideo/data/transcript/TranscriptEvent.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  TranscriptEvent.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@objc public protocol TranscriptEvent {}
```

Comment:
TranscriptEvent is used for parent protocol, Is it intended to have no definition in it?

Reply:
Yes this is intended to be an empty protocol so that we can have a common type to represent data of either `TranscriptionStatus` or `Transcript` type, to be aligned with what JS SDK has:
```
type TranscriptEvent = Transcript | TranscriptionStatus;
```